### DISCUSSION TITLE
選課!!

### DISCUSSION THREAD

**Discussion by spermwhale464** [1101-GS4719-C/Python-Community-GS4719-C#1797] [General]
RT
初選期間，有沒有推薦分數高點的課(甚麼課都可以 不限定通識)
想衝分

**Comment by eric202501** [comment]
資料蒐集與報告寫作
生命教育
材料，改變我們的世界
python程式設計 (張家凱老師的課)超棒

**Comment by mrweixd** [comment]
在這邊也卡一個推薦。

**Comment by poiu6fi** [comment]
python程式設計
心理學
運算思維
聽說通識地球物理也很不錯，不過自己沒去上過

**Comment by Tchang323** [comment]
如果對學法文有興趣的話，可以去選修法文，只要上課認真，作業也不多，就能拿高的分數

**Comment by qiu0109** [comment]
對日文有興趣的話，我覺得日本課還不錯，給分蠻甜的，而且可以學到東西，大概上個兩學期就可以去考n4，即使之後沒有要上了也可以自學日文。
